medical/models/medical_patient.py

```diff
@@ -60,110 +34,79 @@ class MedicalPatient(orm.Model):
     form around medical.
     '''
     _name = 'medical.patient'
+    _description = 'Medical Patient'
     _inherits = {'res.partner': 'partner_id', }
 
-    def _get_default_patient_id(self, cr, uid, context=None):
+    @api.model
+    def _get_default_patient_id(self):
         """ Gives default patient_id """
-        patient_ids = self.search(cr, uid, [('name', '=', 'Libre')],
-                                  context=context)
-        if not patient_ids:
-            raise orm.except_orm(_('Error!'), _('No default patient defined'))
-
-        return patient_ids[0]
-
-    def _get_age(self, cr, uid, ids, field_name, arg, context=None):
-        res = {}
-        age = ''
+        domain = [('name', '=', 'Libre')]
```

Comment:
@yvaucher  @JonathanNEMRY  patient 'Libre' seems to be used to generate appointments for a given future period for a given medical.physician. These appointments  are considered as 'free' and thus assignable to a real patient since they are linked to this patient named 'Libre'. The functionality doesn't work and IMO conceptually it's a bad idea to generate some kink of pseudo appointments to be able to manage the agenda of a physician. IMO, we should remove this method and the functionalities using this method to get a clean base for future enhancements.